### DISCUSSION TITLE
Current version of Borg not supporting Zstd

### DISCUSSION THREAD

**Discussion by blogdemoi** [borgbase/vorta#1527] [FAQ]
Hi,

I have been using happily Vorta to backup my Desktop computer (Manjaro latest stable) to Borgbase. Today I received this message in Vorta when I tried to backup: Current version of Borg not supporting Zstd

I have been using Zstd the last months without problems. My Vorta is 0.87 and installed Borg is 1.2.2.

What could be wrong as a few days ago it worked fine and I didn't update anything on my Desktop computer? Can I change the compression method without loosing any backup files if I change the compression method?

Thank you.

**Comment by real-yfprojects** [comment]
> Can I change the compression method without loosing any backup files if I change the compression method?

Yes, you can safely change the compression method. It will only affect new archives, the ones already present in the borg repo are left untouched and will still be accessible.

> What could be wrong as a few days ago it worked fine and I didn't update anything on my Desktop computer?

Please open a bug report (github issue).
